frontend/hub/remotes/RemoteForm.tsx

```diff
@@ -494,18 +579,34 @@ const TranslationLabelHelp = () => {
   );
 };
 
-function RequirementsFile() {
+function RequirementsFile({ isCommunityRemote }: IRequirementsFile) {
   const { t } = useTranslation();
+  const requirementsFile = useWatch({ name: 'requirements_file' }) as string;
+  const signedOnlyInput = useWatch({ name: 'signed_only' }) as boolean;
+  const isRequired =
+    isCommunityRemote && signedOnlyInput && requirementsFile === yamlRequirementsTemplate;
```

Comment:
requirements being required depends only on isCommunityRemote, but not signedOnlyInput or the file being the same as the template, right?

Reply:
Thanks @himdel I will remove the signeOnlyInput logic from here, but the same template is to avoid the template to be saved as default since it will be a required one now.

Reply:
@himdel logic was updated.